src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -95,17 +98,14 @@ const DeleteFreeAccountButton: FC = () => {
     history.push(`/orgs/${org.id}/org-settings/delete`)
   }
 
-  const handleShowWarning = () => {
+  const showRemoveUsersWarning = () => {
     const buttonElement: NotificationButtonElement = onDismiss =>
-      getDeleteAccountWarningButton(`/orgs/${org.id}/members`, onDismiss)
+      OrgUsersLink(`/orgs/${org.id}/members`, onDismiss)
     dispatch(notify(deleteAccountWarning(buttonElement)))
   }
 
-  let handleDeleteAccountFree = handleShowDeleteOverlay
-
-  if (users.length > 1) {
-    handleDeleteAccountFree = handleShowWarning
-  }
+  const handleDeleteAccountFree =
+    users.length > 1 ? showRemoveUsersWarning : showDeleteFreeAccountOverlay
```

Comment:
I know you didn't write this, but does storing the function in a variable and calling that instead of just creating a function that checks the conditions and calls the right function feel a little weird to you?

Reply:
Yeah, it is definitely weird: my edit only made it more concise, but didn't change what was going on. Let me tweak the logic here a bit.